cardano-testnet/src/Testnet/Start/Cardano.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE CPP #-}
```

Comment:
You solution is correct IMO. One annoying thing: depending on `CPP` breaks code formatters (both stylish and fourmolu give up). So from developer experience it is a bit annoying. So can I ask you to create a separate haskell module which uses `CPP` extension and conditionally installs the handler?